statement_test.go

```diff
@@ -488,12 +484,38 @@ func TestGetQueryID(t *testing.T) {
 		} else {
 			t.Fatal("should have been able to cast to Snowflake Error")
 		}
-		if rows != nil {
-			t.Fatal("rows should be empty")
-		}
 		return nil
 	})
 	if err != nil {
 		t.Fatalf("failed to prepare statement. err: %v", err)
 	}
 }
+
+func TestPreparedStatement(t *testing.T) {
```

Comment:
can you add comments so that the reviews can understand what is the purpose of tests and how you are validating?